core/templates/dev/head/css/oppia.css

```diff
@@ -3819,16 +3819,26 @@ md-card.preview-conversation-skin-supplemental-card {
   background-color: #fff;
   cursor: pointer;
   display: inline-block;
-  height: inherit;
+  height: 321px;
   margin: 8px 5.5px;
   padding: 0 0 20px 0;
   position: relative;
   text-align: left;
-  vertical-align: top;
   white-space: normal;
   width: 250px;
 }
 
+.background-mask {
```

Comment:
Put this in a style tag in the relevant directive file, since it's local to that file. Prefix it appropriately so it's scoped to only stuff within that file.

Reply:
I didn't get what you wanted to say here @seanlip